examples.ipynb

```diff
@@ -0,0 +1,549 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DataHowLab SDK"
+   ]
+  },
```

Comment:
this is a fantastic notebook, well done! 

often (similarly to the README) comments or codelines a too long and then a scroll bar is introduced, this is very nitpicky yes but perhaps we could at some point limit the no. of lines and then make sure users dont have to scroll anymore looking at the notebook code snippets etc.

Reply:
also do we render this notebook in the static html docs? if not we should, its quite easy to do and this is great!